lib/install.go

```diff
@@ -87,12 +98,14 @@ func install(product Product, tfversion, binPath, installPath, mirrorURL, goarch
 
 	/* If unable to download file from url, exit(1) immediately */
 	if errDownload != nil {
+		releaseLock(lockFile)
 		logger.Fatalf("Error downloading: %s", errDownload)
```

Comment:
If we switched to log.Panic, we could remove the `releaseLock` call and let the defer handle it

Reply:
https://github.com/warrensbox/terraform-switcher/pull/551#discussion_r1976474724